[2025-05-18T12:36:03.373+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model permission to 509
[2025-05-18T12:36:03.374+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model'
[2025-05-18T12:36:03.396+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model permission to 509
[2025-05-18T12:36:03.397+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model'
[2025-05-18T12:36:03.428+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.populate_dimensional_model manual__2025-05-18T12:35:52.490194+00:00 [queued]>
[2025-05-18T12:36:03.434+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.populate_dimensional_model manual__2025-05-18T12:35:52.490194+00:00 [queued]>
[2025-05-18T12:36:03.435+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-18T12:36:03.445+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): populate_dimensional_model> on 2025-05-18 12:35:52.490194+00:00
[2025-05-18T12:36:03.450+0000] {standard_task_runner.py:60} INFO - Started process 230 to run task
[2025-05-18T12:36:03.452+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'populate_dimensional_model', 'manual__2025-05-18T12:35:52.490194+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmpyuadqpu3']
[2025-05-18T12:36:03.454+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask populate_dimensional_model
[2025-05-18T12:36:03.491+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model permission to 509
[2025-05-18T12:36:03.491+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T12:35:52.490194+00:00/task_id=populate_dimensional_model'
[2025-05-18T12:36:03.493+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.populate_dimensional_model manual__2025-05-18T12:35:52.490194+00:00 [running]> on host 12957649f5de
[2025-05-18T12:36:03.662+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='populate_dimensional_model' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T12:35:52.490194+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-18T12:35:52.490194+00:00'
[2025-05-18T12:42:13.728+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-18T12:42:13.737+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=populate_dimensional_model, execution_date=20250518T123552, start_date=20250518T123603, end_date=20250518T124213
[2025-05-18T12:42:13.778+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-18T12:42:13.792+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
